tests/custom_assertions.py

```diff
@@ -0,0 +1,50 @@
+
+class CustomAssertions:
+
+    def assertObject(self, actual, expected):
+        """Assert that a model is equivalent to its dict counterpart.
+
+        Verifies that all properties have been set correctly
+        Verifies that all set properties can be accessed by dot notation
+        """
+
+        # if expected is a dict and actual is some model object do the complicated logic
+        if type(expected) is dict and type(actual) is not dict: 
+            for key, value in expected.items():
+                if type(value) is list:
+                    if len(value) != len(getattr(actual, key)):
```

Comment:
```suggestion
            actual_value = getattr(actual, key)
            for key, expected_value in expected.items():
                if type(expected_value) is list:
                    if len(expected_value) != len(actual_value):
```

You call `getattr(actual, key)` quite a few times. It might be easier to read this function if you declare it as a variable and then reference that throughout the rest of it.